apps/src/code-studio/components/progress/StageProgressNew.story.jsx

```diff
@@ -92,7 +91,7 @@ const unplugged = {
 
 export default storybook => {
   const createStoreForLevels = (levels, currentLevelIndex) => {
-    const store = createStore(combineReducers({progress, stageLock, sections}));
+    const store = createStore(combineReducers({progress, stageLock}));
```

Comment:
I apparently missed some cleanup in that we still have a StageProgress and StageProgressNew stories. Will cleanup in future PR.